src/os/unix/file.cpp

```diff
@@ -310,6 +291,11 @@ bool pws_os::LockFile(const stringT &filename, stringT &locker,
         StringXStream lockerStream;
         if (PWSUtil::loadFile(lock_filename.c_str(), lockerStream)) {
           locker = stringx2std(lockerStream.str());
+
+          if (!PWSUtil::HasValidLockerData(locker)) {
+            locker = _T("");
+            Format(locker, IDSC_INVALIDLOCKER, lock_filename.c_str());
```

Comment:
When the application is started from the command line it gets stuck when 'GetStringBufSize' is executed within 'Format'.
This behavior is not shown when pwsafe is started by the IDE.

Reply:
Resolved by updating 'core.rc2' with the newly introduced string.